src/mobile/apps/fair_info/test/routes.test.js

```diff
@@ -84,58 +88,54 @@ describe("FairInfo routes", function () {
         fabricate("fair_event"),
       ]
 
-      this.res.locals.fair = new Fair(fabricate("fair"))
-      return sinon.stub(Backbone, "sync").yieldsTo("success", fairEvents)
+      res.locals.fair = new Fair(fabricate("fair"))
+      sinon.stub(Backbone, "sync").yieldsTo("success", fairEvents)
     })
 
     afterEach(() => Backbone.sync.restore())
 
-    return it("renders the events page", function () {
-      routes.events(this.req, this.res, this.next)
-      this.res.render.called.should.be.true()
-      this.res.render.args[0][0].should.eql("events")
+    it("renders the events page", function () {
+      routes.events(req, res, next)
+      res.render.called.should.be.true()
+      res.render.args[0][0].should.eql("events")
       // all events are on the same day, check to make sure there are 4 events
-      const { eventsByDay } = this.res.render.args[0][1]
+      const { eventsByDay } = res.render.args[0][1]
       const dayKey = _.first(_.keys(eventsByDay))
-      return eventsByDay[dayKey].length.should.eql(4)
+      eventsByDay[dayKey].length.should.eql(4)
     })
   })
 
   xdescribe("#singleEvent", function () {
     beforeEach(function () {
-      this.res.locals.fair = new Fair(fabricate("fair"))
-      return sinon
-        .stub(Backbone, "sync")
-        .yieldsTo("success", fabricate("fair_event"))
+      res.locals.fair = new Fair(fabricate("fair"))
+      sinon.stub(Backbone, "sync").yieldsTo("success", fabricate("fair_event"))
     })
 
     afterEach(() => Backbone.sync.restore())
 
-    return it("renders the single event page", function () {
-      routes.singleEvent(this.req, this.res, this.next)
-      this.res.render.called.should.be.true()
-      this.res.render.args[0][0].should.eql("event")
-      return this.res.render.args[0][1].event.get("name").should.eql("Welcome")
+    it("renders the single event page", function () {
+      routes.singleEvent(req, res, next)
+      res.render.called.should.be.true()
+      res.render.args[0][0].should.eql("event")
+      res.render.args[0][1].event.get("name").should.eql("Welcome")
     })
   })
 
-  return describe("#infoProgramming", function () {
+  describe("#infoProgramming", function () {
     beforeEach(function () {
-      this.res.locals.fair = new Fair(fabricate("fair"))
-      return sinon
-        .stub(Backbone, "sync")
-        .yieldsTo("success", fabricate("article"))
+      res.locals.fair = new Fair(fabricate("fair"))
+      sinon.stub(Backbone, "sync").yieldsTo("success", fabricate("article"))
     })
 
     afterEach(() => Backbone.sync.restore())
 
-    return xit("renders the article page", function () {
-      routes.infoProgramming(this.req, this.res, this.next)
+    xit("renders the article page", function () {
```

Comment:
Did you want to un-pend this, or leave as-is?

Reply:
I didn't comment out or restore any tests. I would prefer to leave them as-is for this PR.

Reply:
👍